frontstage_api/resources/surveys/generate_eq_url.py

```diff
@@ -32,6 +32,12 @@ def get():
         logger.info('Generating EQ URL', case_id=case_id, party_id=party_id)
 
         case = case_controller.get_case_by_case_id(case_id)
+
+        if case['caseGroup']['caseGroupStatus'] == 'COMPLETE':
+            logger.info('The case group status is complete, opening an EQ is forbidden',
+                        case_id=case_id, party_id=party_id)
+            return make_response(jsonify('This questionnaire has already been completed'), 403)
```

Comment:
The 403 would be good enough. We're passing back text unnecessarily. It's never going to be parsed